src/scripts/worldmap/MapHelper.ts

```diff
@@ -248,6 +245,26 @@ class MapHelper {
         }
     }
 
+    public static accessToShip() {
+        return player.highestRegion() > 0 && TownList[GameConstants.DockTowns[player.region]].isUnlocked();
+    }
+
+    // For moving between already-unlocked regions
+    public static moveToRegion(region: GameConstants.Region) {
+        if (MapHelper.accessToShip()) {
+            if (region <= player.highestRegion() && region <= GameConstants.MAX_AVAILABLE_REGION && region !== GameConstants.Region.none) {
+                player.region = region;
```

Comment:
Should we move that line to `moveToTown` so we do not have to worry about it anytime we want the player to move to another region ? Or is it just too niche of an occurrence to be worth it ?